build.sbt

```diff
@@ -126,11 +118,8 @@ lazy val rocketConfig = (project in rocketChipDir / "api-config-chipsalliance/bu
   )
 
 lazy val rocketchip = freshProject("rocketchip", rocketChipDir)
-  .sourceDependency(chiselRef, chiselLib)
-  .settings(addCompilerPlugin(chiselPluginLib))
-  .settings(libraryDependencies ++= chiselLibDeps.value)
   .dependsOn(hardfloat, rocketMacros, rocketConfig)
-  .settings(commonSettings)
+  .settings(commonSettings, chiselSettings)
```

Comment:
Nit: I would enforce either doing `settings(common, chisel)` or having the `settings(chisel)` on a different line for all subprojects.